engine/src/main/java/io/zeebe/engine/nwe/container/SubProcessProcessor.java

```diff
@@ -56,37 +147,4 @@ public void onChildTerminated(
       eventSubscriptionBehavior.publishTriggeredEventSubProcess(flowScopeContext);
     }
   }
-
```

Comment:
I think you don't mean to forget the todo on line 129, so here's a reminder :)

Reply:
Ok, I removed the TODO and added a comment on the state issue :)